Misc/NEWS.d/next/Core and Builtins/2018-10-21-17-43-48.bpo-29743.aeCcKR.rst

```diff
@@ -0,0 +1,4 @@
+Raise `ValueError` instead of `OverflowError` in case of a negative
```

Comment:
There are some issues with using the default role. It is better to use ``` :exc:`ValueError` ``` or ``` ``ValueError`` ``` or just remove mark up.